runtime/doc/intro.txt

```diff
@@ -500,8 +500,7 @@ the ":map" command.  The rules are:
 	   <t_kd>		"kd" termcap entry (cursor down key)
 
 If you want to use the full <> notation in Vim, you have to make sure the '<'
-flag is excluded from 'cpoptions' (when 'compatible' is not set, it already is
-by default). >
+flag is excluded from 'cpoptions'.
```

Comment:
Looks like the `>` was removed. You can use this to toggle conceal levels in order see hidden chars like that:
```
nnoremap coc :set conceallevel=<C-R>=&conceallevel == 2 ? 0 : 2<CR><CR>
```